.github/workflows/build_documentation.yml

```diff
@@ -40,8 +40,13 @@ jobs:
         with:
           python-version: 3.8
```

Comment:
```suggestion
          python-version: 3.10 
```

Reply:
@Revathyvenugopal162 thanks for pointing that out :)